include/zephyr/tracing/tracking.h

```diff
@@ -73,6 +73,81 @@ extern struct k_event *_track_list_k_event;
  */
 #define SYS_PORT_TRACK_NEXT(list)((list)->_obj_track_next)
 
+/**
```

Comment:
i see macros being added, but I do not see them being used, can you please elaborate on the usage of thos macros and why we need them?